test/alternator/test_streams.py

```diff
@@ -359,10 +359,8 @@ def test_get_records(dynamodb, dynamodbstreams):
             while iterators:
                 iter = iterators.pop(0)
                 response = dynamodbstreams.get_records(ShardIterator=iter, Limit=1000)
-                next = response['NextShardIterator']
-
-                if next != '':
-                    next_iterators.append(next)
+                if 'NextShardIterator' in response:
+                    next_iterators.append(response['NextShardIterator'])
```

Comment:
I think this change is already included in a patch I sent, maybe you will need to rebase.

Reply:
Sure. But should merge ok.